docs/sources/flow/getting-started/opentelemetry-to-lgtm-stack.md

```diff
@@ -0,0 +1,308 @@
+---
+title: OpenTelemetry to Grafana stack
+weight: 350
+---
+
+# OpenTelemetry to Grafana stack
+
+In the [previous guide][], we learned how to ingest [OpenTelemetry][] data and export it to a different destination. This guide shows you how to send the ingested OpenTelemetry data to a Loki, Mimir, and Tempo stack.
+
+[previous guide]: {{< relref "./collect-opentelemetry-data.md" >}}
+[OpenTelemetry]: https://opentelemetry.io
+
+
+## Components used in this topic
+
+* [loki.write][]
+* [otelcol.auth.basic][]
+* [otelcol.exporter.loki][]
+* [otelcol.exporter.otlp][]
+* [otelcol.exporter.prometheus][]
+* [otelcol.processor.batch][]
+* [otelcol.receiver.otlp][]
+* [prometheus.remote_write][]
+
+[loki.write]: {{< relref "../reference/components/loki.write.md" >}}
+[otelcol.auth.basic]: {{< relref "../reference/components/otelcol.auth.basic.md" >}}
+[otelcol.exporter.loki]: {{< relref "../reference/components/otelcol.exporter.loki.md" >}}
+[otelcol.exporter.otlp]: {{< relref "../reference/components/otelcol.exporter.otlp.md" >}}
+[otelcol.exporter.prometheus]: {{< relref "../reference/components/otelcol.exporter.prometheus.md" >}}
+[otelcol.processor.batch]: {{< relref "../reference/components/otelcol.processor.batch.md" >}}
+[otelcol.receiver.otlp]: {{< relref "../reference/components/otelcol.receiver.otlp.md" >}}
+[prometheus.remote_write]: {{< relref "../reference/components/prometheus.remote_write.md" >}}
+
+## Before you begin
+
+* Ensure that you have basic familiarity with instrumenting applications with
+  OpenTelemetry.
+* Have a set of OpenTelemetry applications ready to push telemetry data to
+  Grafana Agent Flow.
+* Identify where Grafana Agent Flow will write received telemetry data.
+* Be familiar with the concept of [Components][] in Grafana Agent Flow.
+
+[Components]: {{< relref "../concepts/components.md" >}}
+
+## The Pipeline
+
+We start with the config from the previous guide:
+
+```river
+otelcol.receiver.otlp "example" {
+  grpc {
+    endpoint = "127.0.0.1:4317"
+  }
+
+  http {
+    endpoint = "127.0.0.1:4318"
+  }
+
+  output {
+    metrics = [otelcol.processor.batch.example.input]
+    logs    = [otelcol.processor.batch.example.input]
+    traces  = [otelcol.processor.batch.example.input]
+  }
+}
+
+otelcol.processor.batch "example" {
+  output {
+    metrics = [otelcol.exporter.otlp.default.input]
+    logs    = [otelcol.exporter.otlp.default.input]
+    traces  = [otelcol.exporter.otlp.default.input]
+  }
+}
+
+otelcol.exporter.otlp "default" {
+  client {
+    endpoint = "my-otlp-grpc-server:4317"
+  }
+}
+```
+
+This represents the pipeline:
+```
+Metrics, Logs, Traces: OTLP Receiver → batch processor → OTLP Exporter
+```
```

Comment:
Do we really need this? If I understand this right, this is the pipeline before you reconfigure to send to Loki, Mimir, and Tempo?